src/callbacks_step/analysis_dg2d_parallel.jl

```diff
@@ -3,200 +3,209 @@
+        linf_error = convert(eltype(linf_errors), NaN * zero(eltype(linf_errors)))
+    end
 
-  return l2_error, linf_error
+    return l2_error, linf_error
 end
 
 function calc_error_norms_per_element(func, u, t, analyzer,
-                                      mesh::ParallelTreeMesh{2}, equations, initial_condition,
+                                      mesh::ParallelTreeMesh{2}, equations,
+                                      initial_condition,
                                       dg::DGSEM, cache, cache_analysis)
-  @unpack vandermonde, weights = analyzer
-  @unpack node_coordinates = cache.elements
-  @unpack u_local, u_tmp1, x_local, x_tmp1 = cache_analysis
-
-  # Set up data structures
-  T = typeof(zero(func(get_node_vars(u, equations, dg, 1, 1, 1), equations)))
-  l2_errors = zeros(T, nelements(dg, cache))
-  linf_errors = copy(l2_errors)
-
-  # Iterate over all elements for error calculations
-  for element in eachelement(dg, cache)
-    # Interpolate solution and node locations to analysis nodes
-    multiply_dimensionwise!(u_local, vandermonde, view(u,                :, :, :, element), u_tmp1)
-    multiply_dimensionwise!(x_local, vandermonde, view(node_coordinates, :, :, :, element), x_tmp1)
-
-    # Calculate errors at each analysis node
-    volume_jacobian_ = volume_jacobian(element, mesh, cache)
-
-    for j in eachnode(analyzer), i in eachnode(analyzer)
-      u_exact = initial_condition(get_node_coords(x_local, equations, dg, i, j), t, equations)
-      diff = func(u_exact, equations) - func(get_node_vars(u_local, equations, dg, i, j), equations)
-      l2_errors[element] += diff.^2 * (weights[i] * weights[j] * volume_jacobian_)
-      linf_errors[element] = @. max(linf_errors[element], abs(diff))
+    @unpack vandermonde, weights = analyzer
+    @unpack node_coordinates = cache.elements
+    @unpack u_local, u_tmp1, x_local, x_tmp1 = cache_analysis
+
+    # Set up data structures
+    T = typeof(zero(func(get_node_vars(u, equations, dg, 1, 1, 1), equations)))
+    l2_errors = zeros(T, nelements(dg, cache))
+    linf_errors = copy(l2_errors)
+
+    # Iterate over all elements for error calculations
+    for element in eachelement(dg, cache)
+        # Interpolate solution and node locations to analysis nodes
+        multiply_dimensionwise!(u_local, vandermonde, view(u, :, :, :, element), u_tmp1)
+        multiply_dimensionwise!(x_local, vandermonde,
+                                view(node_coordinates, :, :, :, element), x_tmp1)
+
+        # Calculate errors at each analysis node
+        volume_jacobian_ = volume_jacobian(element, mesh, cache)
+
+        for j in eachnode(analyzer), i in eachnode(analyzer)
+            u_exact = initial_condition(get_node_coords(x_local, equations, dg, i, j),
+                                        t, equations)
+            diff = func(u_exact, equations) -
+                   func(get_node_vars(u_local, equations, dg, i, j), equations)
```

Comment:
That's something for @sloede: IIRC, you insisted on adding parentheses instead of using implicit line continuation after binary operators. DO you want o check everything to impose this restriction? I'm fine either way - we could also keep this in mind whenever we update some code in the future

Reply:
I know... unfortunately, the formatter is not smart enough to detect this and add parentheses itself 😢 I think we have to live with this being in the code, and I propose to adopt the habit of adding parens whenever we see this somewhere in the code...

Reply:
Sounds good to me